src/pixi/renderers/webgl/utils/WebGLShaderManager.js

```diff
@@ -67,6 +67,10 @@ PIXI.WebGLShaderManager.prototype.setContext = function(gl)
     // the next one is used for rendering triangle strips
     this.stripShader = new PIXI.StripShader(gl);
 
+
+    // the next one is used for rendering creature meshes
+    this.creatureShader = new PIXI.CreatureShader(gl);
```

Comment:
Hi @andrewjb123 this line will error in a build without Creature. Can you make a new PR with a fix? Thanks.